Bottom: 235cf0c788d32f5742914ca0e61448bcd2cdf205
Top:    25d350c655447e0d5e6f2b429b9034e88ec50045
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-03-11 19:11:24 +0000

Refresh of remove-settings-init_done

---

diff --git a/src/global.h b/src/global.h
index 19f4454..80bea72 100644
--- a/src/global.h
+++ b/src/global.h
@@ -668,7 +668,6 @@ typedef struct			/* some resources... */
     /* connections is done */
     AutoPlace;			/* flag which says we should force placement of the
 				   windows on startup */
-  int init_done;
 }
 SettingType, *SettingTypePtr;
